src/cddl-test/Main.hs

```diff
@@ -107,6 +110,11 @@ dbArgs fp = do
     , immCheckIntegrity = const True -- No validation
     , immHasFS          = ioHasFS $ MountPoint (fp </> "immutable")
     , immTracer         = nullTracer
+    , immCacheConfig    = CacheConfig
+        { pastEpochsToCache = 1
+        , expireUnusedAfter = 42
```

Comment:
Assuming these magic values are ok for a test. Worst thing than can happen is a test fails.